recipe/meta.yaml

```diff
@@ -228,7 +243,7 @@ outputs:
         - wrapt >=1.11.0
         # TF-API needs to move in sync
         - tensorboard >=2.17,<2.18
-        - keras >=3.0
+        - keras >=3.2
       run:
         - python
         - packaging
```

Comment:
Is `numpy` required at runtime too? See https://github.com/tensorflow/tensorflow/blob/v2.17.0/tensorflow/tools/pip_package/setup.py#L88-L89

Reply:
Do we need `pyyaml` on `linux`? Or is it the Ubuntu-specific dependency? See https://github.com/tensorflow/tensorflow/blob/v2.17.0/tensorflow/tools/pip_package/setup.py#L118

Reply:
It seems python 3.11 for bazel 6 requires `lit`, see https://github.com/tensorflow/tensorflow/blob/v2.17.0/tensorflow/tools/ci_build/release/requirements_common.txt#L45

Reply:
Is `certifi` required on `osx`? see https://github.com/tensorflow/tensorflow/blob/v2.17.0/tensorflow/tools/ci_build/release/requirements_common.txt#L45

Reply:
Yes we do need numpy. It was there through transitive dependencies but I will add it explicitly. Thanks!

pyyaml is for tests only. 

We can do without certifi and lit. What the build does is pull from, for example, https://github.com/tensorflow/tensorflow/blob/v2.17.0/requirements_lock_3_11.txt